docs/account-management/connecting-github-and-semaphore.md

```diff
@@ -0,0 +1,100 @@
+---
+description: This document provides instructions on how to connect your GitHub and Semaphore 2.0.
+---
+
+# Connecting your GitHub account and Semaphore
+
+### Signing up on Semaphore with GitHub account
+In order to support single sign-on via GitHub Semaphore uses [OAuth app](https://github.com/settings/connections/applications/328c742132e5407abd7d).
+
+Semaphore requires read access to your GitHub email address during the sign-up process. 
+
+Once you sign up, there are two different ways to give access to your repositories - through your _personal access token_ or the _GitHub App instalation_.
+
+### OAuth App vs. GitHub App
+The difference between OAuth Apps and GitHub Apps is described in [GitHub documentation](https://docs.github.com/en/developers/apps/about-apps):
+
+> By default, only organization owners can manage the settings of GitHub Apps in an organization. To allow additional users to manage GitHub Apps in an organization, an owner can grant them GitHub App manager permissions.
+> 
+> By contrast, users authorize OAuth Apps, which gives the app the ability to act as an authenticated user. For example, you can authorize an OAuth App that finds all notifications for the authenticated user. You can always revoke permissions from an OAuth App.
+
+Connection through OAuth app relies on your personal access token while with GitHub App the communication goes through GitHub App installation.
+
+It is advised that you use GitHub App as the method of connecting your repositories because of the two main advantages:
+
+- **More granular permission** - Using GitHub App allows you to give access to each repository individually.
+- **No OAuth token owner** - GitHub app doesn't rely on the personal access token of the original Semaphore project owner, making the user offboarding easier. 
+
+### Connecting a repository via GitHub App
+In order to connect the repository via [Semaphore GitHub App](https://github.com/apps/semaphore-ci-cd) three conditions need to be met:
+1. You are a collaborator on the repository.
+2. You installed the GitHub App on the GitHub organization/account that the repository belongs to.
+3. You gave the GitHub App access either to all repositories or the one you want to connect to.
+
+If the conditions above are not met, you will see the empty repository list when trying to create the project:
+
+![GH App - Empty list](.images/ghapp_zero.png)
+
+**To give access to more repositories:**
+
+**Step 1.** Click on the "Give access to more repositories"  
+**Step 2.** Select GitHub account/organization on which you want to install the app  
+**Step 3.** Select "All repositories" or pick individual ones you want to give access to.   
+**Step 4.** You will be returned to the repository list in the Semaphore and repositories should appear.   
+
+You can always edit and update the access permissions of each installed GitHub App. 
+
+!!! info "Installing the GitHub App on GitHub organization"
```

Comment:
Instead of marking the important info with !!! you can also use
**Important:**

Reply:
Those exclamation points are just part of the markup. Once live, it will render like a blue box on this page: https://docs.semaphoreci.com/ci-cd-environment/custom-ci-cd-environment-with-docker/